web/patient_handlers_test.go

```diff
@@ -37,6 +38,14 @@ func (suite *patientSuite) SetupSuite() {
 	api.GET("/patients/:id", web.Adapt(web.GetPatient, suite.withTestService()))
 }
 
+func generateBirthdate(birthdate string) *models.FHIRDateTime {
+	date, err := time.ParseInLocation("2006-01-02", birthdate, time.Local)
+	if err != nil {
+		panic(err)
```

Comment:
log.Fatalln(err) would be better here. panics generate a stack trace which can be annoying when we just need the error that happened and for the app to shut down.